airbyte-api/src/main/openapi/config.yaml

```diff
@@ -995,6 +997,34 @@ paths:
                 $ref: "#/components/schemas/HealthCheckRead"
         "404":
           description: Debug info not found
+  /v1/migration/export:
+    post:
+      tags:
+        - migration
+      summary: Export Airbyte Configuration and Data
+      operationId: exportAirbyte
+      responses:
+        "200":
+          description: Successful operation
+          content:
+            application/x-gzip:
+              schema:
+                $ref: "#/components/schemas/MigrationArchive"
+  /v1/migration/import:
+    post:
+      tags:
+        - migration
+      summary: Import Airbyte Configuration and Data
+      operationId: importAirbyte
+      requestBody:
+        content:
+          application/x-gzip:
+            schema:
+              $ref: "#/components/schemas/MigrationArchive"
+        required: true
+      responses:
```

Comment:
can we make it return an json response object? can be an empty object for now but this way we have an homogeneous API and we might need in the future to return some status